Cargo.lock

```diff
@@ -1054,6 +1094,24 @@ dependencies = [
  "vm",
 ]
 
+[[package]]
+name = "enr"
+version = "0.1.0-alpha-1"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "43184e030b34fb9e8b8ec5a182183855bcf2e07c90f7965ce5a289dcbf9e48ae"
+dependencies = [
+ "base64 0.10.1",
+ "bs58",
+ "clap",
+ "ed25519-dalek",
+ "hex",
```

Comment:
This [hex](https://crates.io/crates/hex) dependency is also quite bad because we already have [rustc-hex](https://crates.io/crates/rustc-hex) in the dependency tree.

IIRC, both do the same but [hex](https://crates.io/crates/hex) also works for slices so we might want to replace `rustc-hex` with `hex` but we should not have both.

Reply:
If so, would it be possible to have both for now? Porting from `rustc-hex` clearly falls outside the scope of this PR.

Reply:
I think we should remove `rustc-hex` in favor of `hex` in the long run. For now, I think it is fine to have both. So let's keep it out of this PR if not anyone has strong opinions.

/cc @dvdplm @ordian

Reply:
we're using `rustc-hex` in a lot of crates in parity-common, https://github.com/paritytech/parity-common/search?q=rustc-hex&unscoped_q=rustc-hex. I'm not sure how easy it would be to get rid of it completely, but def out of scope of this PR

Reply:
It’s a big chunk of (boring) work and an effort better spent doing something else imo. So my advice is we stick to rustc-hex here until someone’s bored enough to tackle the upgrade. 
Sadly I think we still have two versions of rustic-hex in the tree so it’d be even sadder to end up with 3 crates doing the same job.

Reply:
It’s a big chunk of (boring) work and an effort better spent doing something else imo. So my advice is we stick to rustc-hex here until someone’s bored enough to tackle the upgrade. 
Sadly I think we still have two versions of rustic-hex in the tree so it’d be even sadder to end up with 3 crates doing the same job.

Reply:
You're probably right David.